website/src/routes/guides/(migration)/migrate-from-zod/index.mdx

```diff
@@ -1,7 +1,101 @@
 ---
 title: Migrate from Zod
+description: Im based on Zod, so migrating should not require much work, because the naming and behaviour of methods is very similar.
```

Comment:
typo? Im -> I'm
But I am not sure whether it's true that "Valibot is based on Zod" 🤔 I am sure it's inspired by Zod at least though